executor/adapter.go

```diff
@@ -747,6 +747,13 @@ func (a *ExecStmt) buildExecutor() (Executor, error) {
 	}
 
 	b := newExecutorBuilder(ctx, a.InfoSchema)
+
+	if pg, ok := a.Plan.(*plannercore.PointGetPlan); ok {
+		pg.TrackMem = true
+	}
+	if bpg, ok := a.Plan.(*plannercore.BatchPointGetPlan); ok {
+		bpg.TrackMem = true
+	}
```

Comment:
Why we need the field `TrackMem`, and when the value will be false?

Reply:
I’m not sure PointGetExec need track memory when it have parent Executor。so only track memory when The PointGetExec don't have parent executor.

Reply:
I’m not sure PointGetExec need track memory when it have parent Executor。so only track memory when The PointGetExec don't have parent executor.

Reply:
Why tracking memory is related to having parent Executor or not ?
I think we don't need the `TrackMem`, but just do it.